string name
uint32 msg_id
BitMaskData[] bit_masks
uint16 freq
float64 offset
float64 scale
string comment
